task/upload.go

```diff
@@ -146,7 +147,14 @@ func TaskTranscodeFile(tctx *TaskContext) (*TaskHandlerOutput, error) {
 		},
 		finalize: func(callback *clients.CatalystCallback) (*TaskHandlerOutput, error) {
 			tctx.Progress.Set(1)
-			return &TaskHandlerOutput{}, nil
+			if callback.Outputs == nil || len(callback.Outputs) < 1 {
+				return nil, fmt.Errorf("invalid video outputs: %v", callback.Outputs)
+			}
+			return &TaskHandlerOutput{TaskOutput: &data.TaskOutput{
+				TranscodeFile: &data.TranscodeFileTaskOutput{
+					VideoFilePath: clients.RedactURL(callback.Outputs[0].Manifest),
```

Comment:
Just to be sure, this will normally just be a public `ipfs://<cid>` URL right?

Reply:
Depending on the output storage system:
- For Storj => Redacted S3 URL
- For Web3.storage => Web3.storage gateway public URL

No sure, we should return it in the `ipfs://<cid>` format. I'll clarify it with Yondon.

Reply:
Yeah, makes sense. Since it's web3.storage specific now, I think it's ok to return the URL of the w3s gateway for now.

Reply:
Yeah agreed, especially since I just saw the corresponding code has already been merged. No need to bother about this now, let's keep as an idea for the future